ruby/inc/leatherman/ruby/api.hpp

```diff
@@ -209,11 +213,11 @@ namespace leatherman {  namespace ruby {
         /**
          * See MRI documentation.
          */
-        VALUE (* const rb_num2ulong)(VALUE);
+        LONG_LONG (* const rb_num2ull)(VALUE);
```

Comment:
Missed it, should be `unsigned LONG_LONG`, https://github.com/ruby/ruby/blob/trunk/include/ruby/ruby.h#L724